components/Tabs2/tests/unit/MDCTabBarViewTests.m

```diff
@@ -19,10 +19,103 @@
 @interface MDCTabBarViewTests : XCTestCase
 @end
 
-@implementation MDCTabBarViewTests
+@implementation MDCTabBarViewTests {
+  MDCTabBarView *_tabBarView;
```

Comment:
Nit: Please prefer properties over instance variables. 

```objc
@interface MDCTabBarViewTests
@property(nonatomic, strong) MDCTabBarView *tabBarView;
...
```

Reply:
Fix applied.

Reply:
Looks like these are still instance variables.  I'm going to unresolve this to track the outcome.

Reply:
Fixed.